app/views/dictionaries/index.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 Listing dictionaries
```

Comment:
Probably this file was auto generated and you used it for testing. If we don't use it in user interface - delete it